File: tests/fixtures/positive/security/StaticIv.java
package test;

import javax.crypto.spec.GCMParameterSpec;
import javax.crypto.spec.IvParameterSpec;

class Crypto {
    void params() {
        new IvParameterSpec(new byte[] {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0});
        new GCMParameterSpec(128, "000000000000".getBytes());
    }
}

File: tests/fixtures/positive/security/StaticIv.kt
package test

import javax.crypto.spec.GCMParameterSpec
import javax.crypto.spec.IvParameterSpec

class Crypto {
    fun params() {
        IvParameterSpec(byteArrayOf(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
        GCMParameterSpec(128, "000000000000".toByteArray())
    }
}
